docs/apm-breaking.asciidoc

```diff
@@ -5,14 +5,70 @@
 === Breaking Changes
 
 // These tagged regions are required for the stack-docs repo includes
-// tag::83-bc[]
-// end::83-bc[]
+// tag::87-bc[]
+// end::87-bc[]
 // tag::notable-v8-breaking-changes[]
 // end::notable-v8-breaking-changes[]
 
 This section describes the breaking changes and deprecations introduced in this release
 and previous minor versions.
 
+// tag::86-bc[]
+[float]
+[[breaking-changes-8.6]]
+=== 8.6
+
+The following breaking changes and deprecations are introduced in APM version 8.6.0:
+
+[float]
+==== `apm-server.decoder.*` no longer recorded
+The stack monitoring metrics, `apm-server.decoder.*`, are no longer recorded.
+These metrics were not used by stack monitoring, so there should be no noticeable change.
+
+For more details, see https://github.com/elastic/apm-server/pull/9210[PR #9210].
+
+[float]
+==== `ecs.version` is now a `constant_keyword` field
+Instead of adding `ecs.version` as a field in every document written by APM Server,
+it is now a `constant_keyword` field with its value set in data stream mappings.
+
+This change removes `ecs.version` from `_source`. It is now only available through
+the {ref}/script-fields-api.html[{es} `field` API].
+APM does not currently rely on the `ecs.version` field, so there should be no noticeable change.
+
+For more details, see https://github.com/elastic/apm-server/pull/9208[PR #9208].
```

Comment:
IMO this can really be considered a technical detail, not sure how this would imact anyone.

Reply:
Perfect. I'll remove it.